spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Linear.scala

```diff
@@ -81,6 +81,12 @@ class Linear[T: ClassTag](
     zeroGradParameters()
   }
 
+  override def computeOutputShape(inputShape: Activity): Activity = {
+    val input = inputShape.toTensor[Int].toArray()
```

Comment:
what if the inputShape tensor storage offset is not 0

Reply:
We should suppose such behavior to simplify the logic or put this into method doc?

Reply:
It's not safe. Instead you should create a shape class instead of using tensor